src/apps/explorer/components/OrderWidget/OrderDetails.tsx

```diff
@@ -0,0 +1,111 @@
+import React, { useMemo } from 'react'
+import styled from 'styled-components'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faQuestionCircle } from '@fortawesome/free-solid-svg-icons'
+
+import { RawOrder } from 'api/operator'
+
+import { SimpleTable } from 'components/common/SimpleTable'
+
+const COLUMN_COLOR = '#252535'
```

Comment:
please use the theme :)

Reply:
Yes, will get there :)

Forgot to leave a TODO indicating I should do this. Will update.

Reply:
cool cool